pom.xml

```diff
@@ -281,7 +281,7 @@
     <version.rome>1.0</version.rome>
     <version.tomcat>9.0.22</version.tomcat>
     <version.wsdl4j>1.6.3</version.wsdl4j>
-    <version.xerces>2.12.0.SP04</version.xerces>
+    <version.xerces>2.12.0.SP04-redhat-00001</version.xerces>
```

Comment:
we should not have internal built artifacts in sources.
Why? it is not published in public repository, only in Red Hat product one.